plop-templates/core/Story.js.hbs

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import {{componentName}} from '../src/{{componentName}}'
+
+storiesOf('{{componentName}}', module).add('default', () => (
+    <{{componentName}} color='primary.base'>
```

Comment:
`color` should only be `primary`, it will use the `${getPaletteColor('base')}` from the component so it will already be `primary.base`

Reply:
I forget what the exact reason was, but @bertya gave an explanation yesterday as to why keeping `base` is preferable.